airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -310,6 +325,29 @@ else if (hasFailed.get()) {
 
   }
 
+  private class ThreadedTimeHolder {
+
+    private long sourceReadEndTime = -1;
+    private long destinationWriteEndTime = -1;
```

Comment:
nit: since this class is private and only used within the `DefaultReplicationWorker` class it's probably fine to drop the `private` access modifiers and simplify by removing the various getter methods and directly access the variable.

OOP proponents will probably spit at me - I always opt for less LOC if possible, especially if it's a POJO, which this pretty much is!